pkg/server/commands/listusers/list_users.go

```diff
@@ -2,6 +2,7 @@ package listusers
 
 import (
 	openfgav1 "github.com/openfga/api/proto/openfga/v1"
+	"golang.org/x/exp/maps"
```

Comment:
why do we use `golang.org/x/exp/maps` vs. `maps`?  [Clone](https://pkg.go.dev/maps#Clone) is available as in standard go map